src/sentry/event_manager.py

```diff
@@ -1829,13 +1830,10 @@ def _create_group(project: Project, event: Event, **group_creation_kwargs: Any)
     # add sdk tag to metadata
     group_data.setdefault("metadata", {}).update(sdk_metadata_from_event(event))
 
-    # Add severity to metadata for alert filtering for errors events.
-    # We can skip this if the group type is explicitly NOT an error.
+    # add severity to metadata for alert filtering
     group_type = group_creation_kwargs.get("type", None)
-    severity: Mapping[str, Any] = {}
-    if not group_type or group_type == ErrorGroupType.type_id:
```

Comment:
@snigdhas should this code be removed?

Reply:
yep, I've moved it into `_get_severity_metadata_for_group ` to keep severity-logic out of `_create_group `